### PR TITLE
refactor: use `recolouring` theme without feature flag evaluation

### PR DESCRIPTION
**Pull request by kark**

### CODE REVIEW

File: packages/application-shell/src/components/application-shell-authenticated/application-shell-authenticated.tsx [hunk 1]
```diff
@@ -48,6 +48,7 @@ import VersionTracker from '../version-tracker';
 type TApplicationShellAuthenticationProps = {
   featureFlags?: TFlags;
   defaultFeatureFlags?: TFlags;
+  /** @deprecated */
```
Comment (CarlosCortizasCT) [inline packages/application-shell/src/components/application-shell-authenticated/application-shell-authenticated.tsx:51]: We can safely remove this one.
It was created as a escape hatch we didn't use.

**Comment by CarlosCortizasCT** [comment]
The app-kit components VRTs seem to still use the previous theme. Can we change it to the new one?

image

**Comment by kark** [comment]
> The app-kit components VRTs seem to still use the previous theme. Can we change it to the new one?
> 
Thanks, good point 👍 Updated in [83850babd94308a1d0a59d9e9f908dbd1b6e1366](https://github.com/commercetools/merchant-center-application-kit/pull/3474/commits/8c69a95d635599f29a8eb85b3b0bbd94b1e50ad3)

**Review by FilPob — APPROVED** [review]

**Review by CarlosCortizasCT — APPROVED** [review]
Many thanks, Kacper 🙇

File: packages/application-shell/src/components/authenticated/oidc-callback-error-page.tsx [hunk 2]
```diff
@@ -51,8 +51,8 @@ const AuthCallbackErrorPage = (props: TProps) => {
       {({ locale, messages }) => (
         <ConfigureIntlProvider locale={locale} messages={messages}>
           <ThemeProvider
-            theme="default"
-            themeOverrides={themesOverrides.default}
+            theme="recolouring"
```
Comment (emmenko) [inline packages/application-shell/src/components/authenticated/oidc-callback-error-page.tsx:54]: Should we change it first in UI Kit? Then we can keep this as `default`.

**Review by emmenko — COMMENTED** [review]
Thanks!

File: packages/cypress/src/add-commands/login.ts [hunk 3]
```diff
@@ -279,10 +279,11 @@ function loginByOidc(
               STORAGE_KEYS.SESSION_SCOPE,
               sessionScope
             );
-            // Disable redesign announcement
+            // To hide the dialog for the rebranding colors.
+            // TODO: remove once the dialog has been removed from App Kit.
             win.localStorage.setItem(
-              'isNewDesignReleaseNotificationClosed',
-              'true'
+              'isRecolouringNotificationClosed',
```
Comment (ddouglasz) [inline packages/cypress/src/add-commands/login.ts:285]: Thanks! 💯

**Review by ddouglasz — APPROVED** [review]
Thank you!

File: packages/application-shell/src/components/authenticated/oidc-callback-error-page.tsx (hunk 2 above)
Comment (kark) [reply packages/application-shell/src/components/authenticated/oidc-callback-error-page.tsx:54]: Are we good with proceeding this way and doing the clean up as a follow up?

Comment (emmenko) [reply packages/application-shell/src/components/authenticated/oidc-callback-error-page.tsx:54]: Clarified in Slack. All good

**Review by emmenko — APPROVED** [review]

**Pull request merged**